### DISCUSSION TITLE
Payout

### DISCUSSION THREAD

**Discussion by a1b2d56** [community/community#61540] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

When would payouts be initiated?
My github username is @a1b2d56
Stripe is showing money but havent received yet.
Any help would be appreciated ..
![Screenshot_2023-07-23-09-40-03-41_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/community/community/assets/81574768/c11a5cdc-7b9e-4a0f-b49b-ce13d5ad28ea)

**Comment by antn** [comment, marked as answer]
Hi @a1b2d56! I'm from the GitHub Sponsors team, and I looked at your account to see what might be going on.

It looks like it hasn't been 60 days since you've received your first sponsorship. You can see this 60 day waiting period in our terms: https://docs.github.com/en/site-policy/github-terms/github-sponsors-additional-terms#43-payment-timing

> For first payments, GitHub will remit to you all Sponsored Developer Payments 60 days after the first sponsorship is initiated, subject to the variables below. This probation period is applied to all Sponsored Developer Accounts and helps insure the integrity of the program.

You should be eligible for payouts starting on July 31st, 2023 (60 days since your first sponsorship). Payouts are sent on the 22nd of every month, so that means that you should receive a payout on August 22, 2023. Hope that helps!

**Reply by a1b2d56** [reply]
Thank you!
